README.md

```diff
@@ -63,6 +63,45 @@ public void testAssertOnUnhandledException() throws InterruptedException {
 }
```

Comment:
Add rule.clearException to end of the test:

 // Clear the exception for passing the running unit test
 rule.clearException();